backend/src/main/java/com/example/backend/model/PetDTO.java

```diff
@@ -0,0 +1,38 @@
+package com.example.backend.model;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+public record PetDTO(String name,
+                     String nameOfBreed,
+                     String photo,
+                     String[] supplies) {
```

Comment:
You can use a List and than you dont neet to override methods ;-)
```suggestion
                     List<String> supplies) {
```

Reply:
Hallo Marcell. Danke fürs Review. Ich wollte nicht zu viele Zeilen review lassen und obwohl ich 100% test Coverage bei Service und Controller hatte, hat SonarCloud gefordert Pet und PetDTO zu testen um diese Mindest-Test-Coverage zu erreichen. Danke dir für das Review :-)